entity-framework/core/modeling/bulk-configuration.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Model Bulk Configuration - EF Core
+description: How to apply bulk configuration during model building in Entity Framework Core
+author: AndriySvyryd
+ms.date: 11/05/2021
+uid: core/modeling/bulk-configuration
+---
+# Model bulk configuration
+
+When an aspect needs to be configured in the same way across multiple entity types the following techniques allow to reduce code duplication and consolidate the logic.
+
+See the [full sample project](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Modeling/BulkConfiguration) containing the code snippets presented below.
+
+## Metadata API
+
+Every builder object returned from <xref:Microsoft.EntityFrameworkCore.ModelBuilder> exposes a <xref:Microsoft.EntityFrameworkCore.ModelBuilder.Model> or `Metadata` property that provides a low-level access to the objects that comprise the model. In particular, there are methods that allow you to iterate over specific objects in the model and apply common configuration to them.
```

Comment:
Is it important to discuss `Metadata` and low-level access? The section seems to simply be about bulk configuration in OnModelCreating via foreach, no?

Reply:
Yes, this is about how to get something to iterate on.